go/vt/binlog/binlogplayer/dbclient.go

```diff
@@ -129,10 +130,14 @@ func LogError(msg string, err error) {
 
 // LimitString truncates string to specified size
 func LimitString(s string, limit int) string {
-	if len(s) > limit {
+	ls, err := textutil.TruncateText(s, limit, textutil.TruncationLocationMiddle, "...")
+	if err != nil { // Fallback to simple truncation
+		if len(s) <= limit {
```

Comment:
should we do this check before calling `TruncateText`? we'd potentially save some computation

Reply:
The util function does that, which is why I didn't. But I could do it first here too....

Reply:
Ah that works for me! Didn't know that